src/app/loadout-drawer/loadout-drawer-reducer.ts

```diff
@@ -247,18 +247,29 @@ export function equipItem(
 }
 
 export function applySocketOverrides(
-  { loadoutItem: searchLoadoutItem }: ResolvedLoadoutItem,
+  resolvedLoadoutItem: ResolvedLoadoutItem,
   socketOverrides: SocketOverrides
 ): LoadoutUpdateFunction {
   return produce((draftLoadout) => {
+    const { loadoutItem, item } = resolvedLoadoutItem;
+
+    // We don't actually want to save empty fragment overrides, these are populated when the initial
```

Comment:
Having this function called `applySocketOverrides` is pretty confusing since it doesn't apply them. So what I'm getting from this comment is we're filtering which fragments we save into the loadout via `socketOverrides`?